releasenotes/notes/fix-azure-generators-serialization-18fcdc9cbcb3732e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixes the (de)serialization of Azure generators.
```

Comment:
Nit: this is actually a consequence of the class not being decorated hence not being added to the "components registry", the component wasn't usable regardless its serialization capabilities

Reply:
updated - I realised now, with your comment, that without the `@component` those generators couldn't be added to a pipeline.